packages/TimePicker/src/TimePicker.js

```diff
@@ -11,120 +11,169 @@
 
 import React from "react";
 import PropTypes from "prop-types";
+import SvgClockTime from "@paprika/icon/src/ClockTime";
 import Input from "@paprika/input";
-import Picker from "./Picker/Picker";
+import L10n from "@paprika/l10n";
+import Popover from "@paprika/popover";
+import Picker from "./components/Picker/Picker";
 import TimeInterpreter from "./TimeInterpreter";
-
-import * as scPicker from "./Picker/Picker.styles"; // eslint-disable-line
 import * as sc from "./TimePicker.styles";
 
 const getExplodeTime = TimeInterpreter.parse;
 
-export default class TimeInput extends React.Component {
-  $refInputContainer = React.createRef();
-
-  static propTypes = {
-    ariaLabel: PropTypes.string,
-    defaultIsVisible: PropTypes.bool,
-    defaultValue: PropTypes.string, // it been use on getDerivedStateFromProps which is a useEffect in pre-era of hooks
-    isDisabled: PropTypes.bool,
-    labelAM: PropTypes.string,
-    labelCustom: PropTypes.string,
-    labelError: PropTypes.string,
-    labelHours: PropTypes.string,
-    labelMinutes: PropTypes.string,
-    labelPeriod: PropTypes.string,
-    labelPM: PropTypes.string,
-    onChange: PropTypes.func,
-    prefix: PropTypes.string,
-  };
+const propTypes = {
+  /** Descriptive a11y text for assistive technologies. */
+  a11yText: PropTypes.string,
 
-  static defaultProps = {
-    ariaLabel: "Time (hh:mm)",
-    defaultIsVisible: false,
-    defaultValue: null,
-    labelCustom: "Custom",
-    labelError: "Invalid Format",
-    labelHours: "Hours",
-    labelMinutes: "Minutes",
-    labelPeriod: "Period",
-    labelAM: "am",
-    labelPM: "pm",
-    isDisabled: false,
-    onChange: () => {},
-    prefix: "timePicker",
-  };
+  /** If the TimePicker is set to visible. */
+  defaultIsOpen: PropTypes.bool,
+
+  /** Sets the default value for the TimePicker */
+  defaultValue: PropTypes.string,
+
+  /** If the TimePicker is disabled. */
+  isDisabled: PropTypes.bool,
+
+  /** The label shown for the AM. */
+  labelAM: PropTypes.string,
```

Comment:
these labels, should be on the L10n component, the component was built before L10n exists, so all those props with prefix label can be move to a locale. cc @KDarcey @allison-c